modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -110,7 +110,7 @@ def validate_form_526_submission_claim_date!
 
     def validate_form_526_application_expiration_date!
       return if form_attributes['applicationExpirationDate'].blank?
-      return if Date.parse(form_attributes['applicationExpirationDate']) >= Time.zone.today
+      return if Date.parse(form_attributes['applicationExpirationDate']) > Time.zone.today
```

Comment:
Should we update this to be a DateTime to match the other one (and (attempt to) solve timezone issues)?

Reply:
Not at this time.
This attribute shouldn't even exist.  Or at least not be exposed to consumers.
If the consumer doesn't send us this attribute, then we set the value to 1 year in the future before we call EVSS.
We've told the new consumer that ran into this problem to stop sending us this value and just let us set the value before we call EVSS.  
If we find that the consumer can't stop sending us this attribute for some reason and we find that it raises timezone related probs, then we can create a new ticket to go handle that situation.